packages/build/webpack/constants.js

```diff
@@ -160,6 +172,8 @@ const TS_CHECKER_DEV_CONFIG = {
                     '@labkey/premium/workflow': [labkeyUIPremiumPath + '/workflow'],
                     '@labkey/premium/storage': [labkeyUIPremiumPath + '/storage'],
                     '@labkey/premium/search': [labkeyUIPremiumPath + '/search]'],
+                    '@labkey/ehr': [labkeyUIEhrPath],
+                    '@labkey/ehr/participanthistory': [labkeyUIEhrPath + '/reports/participanthistory']
```

Comment:
Based on what I am seeing in the ehrModules PR this path seems incorrect. I believe it should be:

```
labkeyUIEhrPath + '/participanthistory'
```

Reply:
fixed thanks